src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
+
+*Prerequisites*
+
+* A Kubernetes cluster with at least 4GB of RAM and RBAC:
+** For Minikube 0.26.0 and higher, use the `minikube start --cpus 2 --memory 4096 --extra-config=apiserver.authorization-mode=RBAC` command.
+** For Minikube 0.25.2 and lower, use the `minikube start --cpus 2 --memory 4096 --extra-config=apiserver.Authorization.Mode=RBAC` command.
+* To deploy role binding, use the `kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default` command.
+* Install the https://github.com/kubernetes/helm/blob/master/docs/install.md[Helm] CLI.
+* Set your default Kubernetes context (this is required to use Helm). In Minikube, this is set automatically. Otherwise, modify the `KUBECONFIG` environment variable and then type `kubectl config use-context <my-context>`.
+* Install `tiller` on your cluster:
+.. Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount]: `kubectl create serviceaccount tiller --namespace kube-system`.
+.. Bind it to the almighty `cluster-admin` role: `kubectl apply -f ./tiller-rbac.yaml`.
+.. Install `tiller`: `helm init --service-account tiller`.
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it, use the `minikube addons enable ingress` command.
+* DNS discovery should be enabled. It is enabled by default in Minikube.
 
 [id="cluster-ip"]
-== Cluster IP
+== Obtaining the cluster IP
 
-* If your cluster is running on MiniKube, simply type `minikube ip` at your terminal
-* If your cluster is in the cloud, obtain the hostname or ip address from your cloud provider
+To obtain the cluster IP:
 
-In production, you should specify a hostname (see https://github.com/eclipse/che/issues/8694[here] why). In case you don’t have a hostname (e.g. during development), and would still want to use a host-based configuration, you can use services such as nip.io or xip.io.
+* If your cluster is running on Minikube, type `minikube ip` on the command line.
+* If your cluster is in the cloud, obtain the hostname or IP address from your cloud provider.
 
-In case you’re specifying a hostname, simply pass it as the value of the `ingressDomain` parameter below.
+In production, you must specify a hostname. For details, see https://github.com/eclipse/che/issues/8694[Add an ability to use host-based routing instead of path-based for k8s infras]. If you do not have a hostname and want to use a host-based configuration, use services such as nip.io or xip.io.
 
-If you must use IP address (e.g. your corporate policy prevents you from using nip.io), you would also have to set `isHostBased` to `false`.
+If specifying a hostname, pass it as the value of the `ingressDomain` parameter in the deployment syntax in the following section.
+
+If using the IP address, for example, your corporate policy prevents you from using nip.io, you must also set `isHostBased` to `false`.
```

Comment:
missing conjunction?

Reply:
Changed to "If using the IP address, for example, when your corporate policy prevents you from using nip.io, you must also set `isHostBased` to `false`.". Better?